util/git/creds.go

```diff
@@ -63,10 +128,26 @@ func NewSSHCreds(sshPrivateKey string, caPath string, insecureIgnoreHostKey bool
 
 type sshPrivateKeyFile string
 
+type authFilePaths []string
+
 func (f sshPrivateKeyFile) Close() error {
 	return os.Remove(string(f))
 }
 
+// Remove a list of files that have been created as temp files while creating
+// HTTPCreds object above.
+func (f authFilePaths) Close() error {
+	var retErr error = nil
+	for _, path := range f {
+		err := os.Remove(path)
+		if err != nil {
+			log.Errorf("HttpsCredsCloser: Could not remove temp file %s: %v", path, err)
```

Comment:
minor - should be `HTTPS` uppercase for Golang conventional name

Reply:
Fixed.